### DISCUSSION TITLE
Support request for Tapo doorbell tapo-d230s1

### DISCUSSION THREAD

**Discussion by shanudominic** [koush/scrypted#1135] [Ideas]
Would like to see if its possible to make tapo-d230s1 camera doorbell with hub work on scrypted? Right now the tapo plugin doesn't allow it to be added as its camera feed is coming through hub and therefore no onvif support.

since this is a recently released device it might not have support by scrypted, but just trying to see if its possible to add support for this device.

I am more than happy to provide information from this hub + camera doorbell to make this work, but at the same same time I know adding support for a new device is not that straight fwd either.

**Comment by nichollier88** [comment]
linking another issue where someone was able to interact with the C420 camera through the H200 hub.
https://github.com/JurajNyiri/pytapo/issues/57#issue-1737685488
